test/utility/test_gptr_monad.cpp

```diff
@@ -27,6 +27,14 @@ TEST(Fmap, GVectorInitialize) {
   pando::deallocateMemory(gvec, 1);
 }
 
+TEST(Fmap, VectorInitialize) {
+  constexpr std::uint64_t SIZE = 10;
+  pando::Vector<std::uint64_t> vec;
+  fmap(vec, initialize, SIZE);
```

Comment:
why is this useful?

Reply:
there is one interface to interact with data now, also all the folks in ece have been using it like this.

Reply:
I see, if people want this that's fine -- personally I really do not like this and find it unreadable